src/misc/getMessageText.ts

```diff
@@ -0,0 +1,16 @@
+import { TranslationService } from "../config/translationService";
+
+/**
+ * Searches the language object (or the current language if that parameter is omitted) for a given message code
+ * @param {string} str The message identifier
+ * @param {string} languageName The language name, may be omitted
```

Comment:
What happens if omitted?

Reply:
That case is explained in the method's description :)